scripts/pestle/helpers/camelCase.js

```diff
@@ -0,0 +1,13 @@
+function camelCase(str) {
```

Comment:
There's a lodash helper that would accomplish this too 😁 

https://www.npmjs.com/package/lodash.camelcase

Reply:
I was able to use it from lodash, learned something new 🎉
But was wondering since I need not only a camelCase but also capitalizing the first letter, is it okay to combine with https://www.npmjs.com/package/lodash.upperfirst and do `upperFirst(camelCase(str))`??